### PR TITLE
[MSA] Workaround to launch files without controllers

### PR DESCRIPTION
**Pull request by vatanaksoytezer**
- Enable compilation of controllers
- Add user warning for hand generating controllers
- Issue a warning instead of raising an exception if the controllers haven't been found

### CODE REVIEW

**Review by henningkayser — APPROVED** [review]

File: moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py [hunk 1]
```diff
@@ -414,7 +415,7 @@ def trajectory_execution(
         else:
             file_path = self._package_path / file_path
 
-        self.__moveit_configs.trajectory_execution.update(load_yaml(file_path))
+            self.__moveit_configs.trajectory_execution.update(load_yaml(file_path))
```
Comment (JafarAbdi) [inline moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py:418]:
```suggestion
        if file_path:
            self.__moveit_configs.trajectory_execution.update(load_yaml(file_path))
```

**Review by JafarAbdi — COMMENTED** [review]
This will not use the file_path set from the other if statement

File: moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py [hunk 2]
```diff
@@ -387,9 +387,10 @@ def trajectory_execution(
             controller_pattern = re.compile("^(.*)_controllers.yaml$")
             possible_names = get_pattern_matches(config_folder, controller_pattern)
             if not possible_names:
-                raise RuntimeError(
-                    "trajectory_execution: `Parameter file_path is undefined "
-                    f"and no matches for {config_folder}/*_controllers.yaml"
+                # Warn the user instead of raising exception
+                logging.warning(
+                    "\x1b[33;20mtrajectory_execution: `Parameter file_path is undefined "
+                    f"and no matches for {config_folder}/*_controllers.yaml\x1b[0m"
```
Comment (AndyZe) [inline moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py:393]:
```suggestion
                    "and no matches for {config_folder}/*_controllers.yaml\x1b[0m"
```

Comment (JafarAbdi) [reply moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py]: I'm pretty sure this's needed, why removing it? Printing the value of `config_folder` should make the warning more helpful.

Comment (AndyZe) [reply moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py]: is the typo `f` needed?  (Unless that's some crazy Python syntax I'm not aware of)

**Review by AndyZe — APPROVED** [review]

**Pull request merged**